wdfn-server/waterdata/templates/macros/components.html

```diff
@@ -153,6 +153,41 @@ <h3>Cooperators</h3>
     </span>
 {%- endmacro %}
 
+{% macro CameraComponent(cameras) -%}
+    <div class="wdfn-component" data-component="cameras">
+        <div class="wdfn-accordion usa-accordion">
+            <h2 class="usa-accordion__heading">
+                <button class="usa-accordion__button"
+                        aria-expanded="true"
+                        aria-controls="camera-container"
+                        ga-on="click" ga-event-category="accordion" ga-event-action="interactionWithCameraAccordion">
+                    Monitoring Camera
+                </button>
+            </h2>
+            <div id="camera-container" class="usa-accordion__content">
+                {% for camera in cameras %}
+                    <h3>{{ camera.name }}</h3>
+                    <p>{{ camera.description }}</p>
+                    <div>
+                        <video hidden class="med-video camera-video" title="Timelapse Video of {{ camera.name }}" controls="" autoplay="">
```

Comment:
Just a question: is there an advantage to using 'controls=""' vs 'controls'? I have seen it both ways.

Reply:
No idea ... I have also seen it both ways. I think I cut and pasted this from the StormSummary page. I think I'll remove the quotes because it seems unnecessary.